lib/availability_status_listener.rb

```diff
@@ -27,6 +27,12 @@ def subscribe_to_availability_status
         process_event(event)
       end
     end
+  rescue Kafka::ConnectionError => e
+    Rails.logger.error("Cannot connect to Kafka cluster #{messaging_client_options[:host]}")
+    unless messaging_client_options[:host] == 'localhost'
```

Comment:
So, is the reason we don't retry if the host is 'localhost' because we assume it's a test scenario?
Also, do the test changes cover this section of code?

Reply:
Yes. It is intended to stop retrying if it is a local test.
I added a test to cover this code.